### DISCUSSION TITLE
关于点击个性推荐后内容TOP鼠标会留在在左上角B站LOGO上导致打开选项卡的建议

### DISCUSSION THREAD

**Discussion by ifdelete** [BewlyBewly/BewlyBewly#685] [Ideas]
当想刷新一下页面内容的时候，需要点击上面几个选项卡
但是第一次点击内容会TOP，这时候鼠标会停留在左上角LOGO上导致会自动打开选项卡

想法：
  把左上角LOGO再往左边移动一点 / 加个配置左上角LOGO鼠标悬浮时不自动展开内容？

**Comment by hakadao** [comment]
不打算處理，切換tab向上是開了搜尋頁模式吧
那邊問題很多我不太想處理了

**Reply by hakadao** [reply]
我覺得滑鼠位置是可以控制的，沒必要調整logo位置吧，logo位置和b站播放頁內容是對齊的其實
